app/src/main/res/values/strings.xml

```diff
@@ -365,4 +365,7 @@
     
     <string name="preferred_player_fetcher_notification_title">Getting info…</string>
     <string name="preferred_player_fetcher_notification_message">"The requested content is loading"</string>
+
+    <!-- Debug Only -->
+    <string name="toggle_leak_canary">Leak Canary</string>
```

Comment:
That's a name, so it shouldn't be translatable.

Reply:
Oops. In this case, perhaps we should change it to "Monitor Memory Leaks"? Or "Monitor Leaks" if that's too long?

Reply:
Yes, that's better. I'd go with the second one. It also helps the user to understand what he's enabling :)

Reply:
Fixed.